### PR TITLE
add installation and usage guideness in README

### PR DESCRIPTION
**Pull request by Hexilee**
Add installation and usage guide.

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
ref https://github.com/PingCAP-QE/Naglfar/pull/69

**Comment by Hexilee** [comment]
The rendered [README](https://github.com/PingCAP-QE/Naglfar/blob/docs/README.md)

File: README.md [hunk 1]
```diff
@@ -1,14 +1,317 @@
...
+testworkflows.naglfar.pingcap.com           
+testworkloads.naglfar.pingcap.com           
+```
+
+## Usage
+
+### Add Host Machine
+
+A host machine is the machine to allocate resources and execute job (cluster and workload).
+The "resources" includes memery, cpu cores and exclusive disks.
```
Comment (mahjonp) [inline README.md:56]:
```suggestion
The "resources" include memory, CPU cores, and exclusive disks.
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,317 @@
...
+  exclusiveDisks:
+    - /dev/nvme0n1
+    - /dev/nvme1n1
+```
+
+You can apply a new machine by `kubectl apply -f <file path>`, then execute `kubectl get machine` to checkout applyment result. 
+
+### Request for Resources
+
+There are tow steps to request resources: create your k8s `Namespace` and add a `TestResouceRequest`.
```
Comment (mahjonp) [inline README.md:84]:
```suggestion
There are two steps to request resources: create your k8s `Namespace` and add a `TestResouceRequest`.
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,317 @@
...
+            mountPath: /disks1
+    - name: n3
+      spec:
+        memory: 5GB
+        cores: 2
+```
+
+The disks maps custom names to exclusive disks, it's unnecessary if you do not need exclusive disks. All resources are allocated automatically, naglfar system will find machines for them.
+
+Now apply this file and checout result by `kubectl get trr test-tidb-cluster -n test-tidb-cluster`.
```
Comment (mahjonp) [inline README.md:126]:
```suggestion
Now apply this file and checkout result by `kubectl get trr test-tidb-cluster -n test-tidb-cluster`.
```

File: README.md [hunk 4]
```diff
@@ -1,14 +1,317 @@
...
+  Ssh Port:       33115
+  State:          ready
+Events:           <none>
+```
+
+Then you can access the tidb endpoint by `10.0.2.213:4000`(docker swarm network) or `172.16.0.1:33113`(host machine network). You can even use ssh to login the root user by (`docker/insecure_key`)[https://github.com/PingCAP-QE/Naglfar/blob/master/docker/insecure_key].
+
+### Apply ProcChaos
+
+The `ProcChaos` means process chaos, you can define some pattern and rules to kill processes once or in period.
```
Comment (mahjonp) [inline README.md:278]:
```suggestion
The `ProcChaos` means process chaos, you can define some patterns and rules to kill processes once or in a period.```

**Review by mahjonp — COMMENTED** [review]
I let Grammarly help me check the doc and find grammar errors, please take a look. Rest LGTM

File: README.md (hunk 1 above)
Comment (Hexilee) [reply README.md]: Fixed

**Review by mahjonp — APPROVED** [review]
LGTM

**Pull request merged**